README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
 Repository for homework 01
+
+#**Sarah Jensen**
```

Comment:
You got the bold formatting correct, but missed the space after the # for the header.